tests/test_ops/test_load_ontology.py

```diff
@@ -0,0 +1,135 @@
+import os
+import pytest
+import tempfile
+import shutil
+from unittest.mock import patch, MagicMock
+from dagster import build_op_context
+from nmdc_runtime.site.resources import mongo_resource
+from nmdc_runtime.site.ops import load_ontology
+
+
+@pytest.fixture
+def client_config():
+    # Print details about the MongoDB configuration for debugging
+    mongo_host = os.getenv("MONGO_HOST")
+    mongo_dbname = os.getenv("MONGO_DBNAME")
+    mongo_username = os.getenv("MONGO_USERNAME")
+    
+    print(f"Test MongoDB connection details:")
+    print(f"- MONGO_HOST: {mongo_host}")
+    print(f"- MONGO_DBNAME: {mongo_dbname}")
+    print(f"- MONGO_USERNAME: {mongo_username}")
```

Comment:
Keep the test logs clean! No printing!

Reply:
there are many `prints` in the existing code base, and some use of the `logging` package directly.  

e.g., in test_endpoints.py
```
    print("materializing alldocs...")
```

18 in test_util.py
3 in test_integrity.py
...

I moved print statements in this test file to root logger `logging.info` statements to mimic other parts of the codebase.  I can imagine we want another PR to make module-level logging a thing.